CHANGELOG.md

```diff
@@ -1,3 +1,97 @@
+### citus v10.1.0 (June 15, 2021) ###
+
+* Drops Postgres 11 support
+
+* Fixes a bug that can cause a crash when DEBUG4 logging is enabled
+
+* Fixes column default coming from a sequence
+
+* Fixes data race in `get_rebalance_progress`
+
+* Fixes error message for local table joins
+
+* Fixes FROM ONLY queries on partitioned tables
+
+* Fixes issues caused by public schema being omitted in queries
+
+* Fixes matview access method change issue
+
+* Fixes nested select query with union bug
+
+* Fixes possible segfaults when using Citus in the middle of an upgrade
+
+* Fixes problems with concurrent calls of `DropMarkedShards`
+
+* Fixes relname null bug when parallel execution
+
+* Fixes shared dependencies that are not resident in a database
+
+* Fixes stale hostnames bug in prep. statements after `master_update_node`
+
+* Fixes using 2PC when it might be necessary
+
+* Adds `shard_count` parameter to `create_distributed_table` function
+
+* Adds support for `ALTER DATABASE OWNER`
+
+* Adds support for temporary columnar tables
+
+* `alter_columnar_table_set` enforces columnar table option constraints
+
+* Checks free disk space before moving a shard
+
+* Closes connection after each shard move
+
+* Continues to remove shards after failure in `DropMarkedShards`
+
+* Deprecates the `citus.replication_model` GUC
+
+* Does smarter cleanup of orphaned shards
+
+* Enables `citus.defer_drop_after_shard_move` by default
+
+* Enforces compiler and linker flags by default
+
+* Errors out if inheriting a distributed table
+
+* Executor refrains from opening extra connections
+
+* Fetches shard size on the fly for the rebalance monitor
+
+* Ignores old placements when disabling or removing a node
+
+* Implements an improvement threshold for shard rebalancer moves
+
+* Improves performance of citus_shards
+
+* Introduces `citus.local_hostname` GUC for connections to the current node
+
+* Introduces three new UDFs for partitioned size calculation
+
+* Makes sure that target node in shard moves is eligible for shard move
+
+* Optimizes partitioned disk size calculation for shard rebalancer
+
+* Prevents connection errors by properly terminating connections
+
+* Prevents users from dropping & truncating known shards
+
+* Prints per connection establishment times when DEBUG4 enabled
+
+* Prints per task execution times when DEBUG4 enabled
+
+* Pushes down VALUES clause as long as not in outer part of a JOIN
+
+* Reduces memory usage for multi-row inserts
+
+* Reduces memory usage while rebalancing
+
+* Refactors size queries with new enum `SizeQueryType`
+
+* Removes limits around long partition names
+
+* Uses `PROCESS_UTILITY_QUERY` in utility calls
```

Comment:
Since we already have those two in 10.0.2, I think we should remove them

Reply:
We discussed offline and decided first one is not the same as the one in 10.0.2